core-metastore/src/main/java/org/zalando/nakadi/repository/db/EventTypeRepository.java

```diff
@@ -102,6 +103,18 @@ public EventType mapRow(final ResultSet rs, final int rowNum) throws SQLExceptio
                 new EventTypeMapper());
     }
 
+    public List<EventType> list(final String[] writers){
+        return jdbcTemplate.query(
+                String.format("SELECT et_event_type_object\n" +
```

Comment:
We should either use `SELECT DISTINCT` or find a way around using set-returning `jsonb_to_recordset()`.

Reply:
I added group by clause to prevent dupes